<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>391</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QCheckBox" name="chkBox_A0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>자장면</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBox_A1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>짬뽕</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_price_A0">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>52</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>5,000원</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_price_A1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>175</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>6,000원</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>451</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QCheckBox" name="chkBox_B0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>탕수육</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBox_B1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>양장피</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBox_B2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>팔보체</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_price_B0">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>12,000원</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_price_B1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>15,000원</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_price_B2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>15,000원</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Btn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>430</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>주문</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>431</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>총원</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_price">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>원</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
